crdb/tx_test.go

```diff
@@ -50,15 +50,18 @@ INSERT INTO d.t (acct, balance) VALUES (1, 100), (2, 100);
 		if err != nil {
 			return
 		}
-		for i, bal := range []*int{&bal1, &bal2} {
-			if !rows.Next() {
-				err = fmt.Errorf("expected two balances; got %d", i)
-				return
-			}
-			if err = rows.Scan(bal); err != nil {
+		defer rows.Close()
```

Comment:
i don't understand what you changed here. can you help?

in particular, it seems that if you tried to scan a third value you'd hit an index out of bounds exception.